docs/app/pages/components/components-sections/timeline/timeline/snippets/app.ts

```diff
@@ -53,7 +59,7 @@ interface TimelineEvent {
     color: string;
     date: Date;
     url: string;
-    id: string;
+    id: number;
```

Comment:
Why the change to number?

Reply:
I changed the code to use 'chance.integer()' to return a random number instead of using the same ID for each ticket. It returns an integer so I changed the type of 'id'.

Reply:
ok, I missed this interface was only part of the doc.